R/pkg/R/DataFrame.R

```diff
@@ -628,8 +628,6 @@ setMethod("repartition",
 #'
 #' @param x A SparkDataFrame
 #' @return A StringRRDD of JSON objects
-#' @family SparkDataFrame functions
```

Comment:
@felixcheung I removed these two lines in toJSON part. Correct me, if I am wrong.